components/bl-data-grid-component/src/index.js

```diff
@@ -18,6 +19,7 @@ export default function DataGridComponent({ component, eventHandlers }) {
     filter,
     sortable,
     floatingFilter,
+    resizable: resizableColumns,
```

Comment:
At link from PR description nothing change for me, looks like columns still not fill all space.

Reply:
i forgot to add the modified code to `develop`